napari/layers/vectors/_tests/test_vectors.py

```diff
@@ -409,6 +410,7 @@ def test_edge_color_map_non_numeric_property():
         layer.edge_color_mode = 'colormap'
 
 
+@pytest.mark.filterwarnings("ignore:elementwise comparis:FutureWarning:numpy")
```

Comment:
I this not fixable?

Reply:
caused at line `layer.edge_color = 'vector_type'`
```pytb
FutureWarning                             Traceback (most recent call last)
<ipython-input-8-702e23f49dcf> in <module>
      1 # switch to color cycle
      2 layer.edge_color_mode = 'cycle'
----> 3 layer.edge_color = 'vector_type'
      4 edge_color_array = transform_color(color_cycle * int((shape[0] / 2)))

~/Dropbox (HMS)/Python/forks/napari/napari/layers/vectors/vectors.py in edge_color(self, edge_color)
    384                 self._edge_color_mode = new_mode
    385             self._edge_color_property = edge_color
--> 386             self.refresh_colors()
    387
    388         else:

~/Dropbox (HMS)/Python/forks/napari/napari/layers/vectors/vectors.py in refresh_colors(self, update_color_mapping)
    438                     # and update_color_mapping==False
    439                     edge_color_cycle_keys = [*self.edge_color_cycle_map]
--> 440                     props_in_map = np.in1d(
    441                         edge_color_properties, edge_color_cycle_keys
    442                     )

<__array_function__ internals> in in1d(*args, **kwargs)

~/miniconda3/envs/napdev/lib/python3.8/site-packages/numpy/lib/arraysetops.py in in1d(ar1, ar2, assume_unique, invert)
    578             mask = np.zeros(len(ar1), dtype=bool)
    579             for a in ar2:
--> 580                 mask |= (ar1 == a)
    581         return mask
    582

FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
```

would prefer that be addressed in a different PR